Tests/Runtime/CmColliderTests.cs

```diff
@@ -44,56 +52,73 @@ public IEnumerator CheckSmoothingTime()
             yield return null;
             Assert.That(originalCamPosition == m_Vcam.State.FinalPosition, Is.True);
 
-            var obstacle = GameObject.CreatePrimitive(PrimitiveType.Cube);
-            obstacle.transform.position = originalCamPosition; // put obstacle in the way
+            var obstacle = CreateObstacle(originalCamPosition);
 
+            yield return null;
+            yield return null;
             yield return null;
             Assert.That(originalCamPosition == m_Vcam.State.FinalPosition, Is.False);
             
             obstacle.transform.position = originalCamPosition + Vector3.left * 100f; // move obstacle out of the way
 
+            yield return null;
+            yield return null;
+            yield return null;
```

Comment:
All these yield return null statements, here and elsewhere in this file, look sketchy.  Why are they here?